app/assets/stylesheets/budget.css

```diff
@@ -0,0 +1,321 @@
+:root {
+  --angels_wings: #A2B38E;
```

Comment:
Would be good to break this big file into smaller topical files, for example for `flashes`, `forms`, `tables`, `navbar`...

Reply:
Great idea, Done in ad-css-styling sha: 12530f8